### PR TITLE
Pulls/2.0/autopublish attached assets

### LINKED ISSUE #5: Publish SiteConfig related images on save
E.g. the `Logo` and `FooterLogo` will not show without first being published, and at current `SiteConfig` is not performing this action. Check the extension that applies these relations also adds an `owns`, and that `SiteConfig` performs a recursive publish (or is `Versioned`) when saved/published.

### PR DESCRIPTION
**Pull request by NightJar**
Fixes #5 by increasing the minimum requirement of framework to resolve most of the issue automatically. CWP 2.0 will be based on 4.1 so this is safe - but as this module relies on functionality added after 4.0 this is made explicit via composer.json.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -8,7 +8,8 @@
     ],
     "license": "BSD-3-Clause",
     "require": {
-        "cwp/cwp": "^2"
+        "cwp/cwp": "^2",
+        "silverstripe/framework": "^4.1"
```
Comment (robbieaverill) [inline composer.json]: Maybe use ^4.1@dev or something for now since this doesn’t exist

File: src/Extensions/CWPSiteConfigExtension.php [hunk 2]
```diff
@@ -298,4 +312,10 @@ protected function addSearchOptions(FieldList $fields)
 
         return $this;
     }
+
+    public function onAfterWrite() {
```
Comment (robbieaverill) [inline src/Extensions/CWPSiteConfigExtension.php]: Phpcs will complain about this

File: composer.json (hunk 1 above)
Comment (NightJar) [reply composer.json]: This doesn't actually seem to be the issue here.

The installation resolves and passes, it's phpcs calling out an issue in `silverstripe/recipe-core`:
https://github.com/silverstripe/recipe-core/pull/16

**Comment by NightJar** [comment]
Relaunched Travis since the merge of silverstripe/recipe-core#16 - the build is now green.

**Pull request merged**